common/trinary/tests/test_ptrits.c

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2019 IOTA Stiftung
+ * https://github.com/iotaledger/entangled
+ *
+ * Refer to the LICENSE file for licensing information
+ */
+
+#include <unity/unity.h>
+
+#include "common/trinary/ptrit.h"
+#include "common/trinary/ptrit_incr.h"
+
+int trits_to_int(size_t n, trit_t const *t) {
+  int s = 0;
```

Comment:
better to use:
https://github.com/iotaledger/entangled/blob/0817986f24417c2931fc2ad6cf4f0a69daccdef1/common/trinary/trit_long.h#L25

(Also, this function is a good candidate for refactoring ;))

Reply:
Ok, didn't notice it